stader-cli/validator/deposit.go

```diff
@@ -148,7 +148,19 @@ func nodeDeposit(c *cli.Context) error {
 		fmt.Println("Validator creation process initiate, please wait...")
 	}
 
-	canNodeDepositResponse, err := staderClient.CanNodeDeposit(baseAmount, utilityAmount, big.NewInt(int64(numValidators)), true)
+	cfg, _, err := staderClient.LoadConfig()
+	if err != nil {
+		return err
+	}
+	canNodeDepositReloadKeysFlag := true
+	nodeDepositReloadKeysFlag := true
+
+	if !cfg.IsSSVMode {
+		canNodeDepositReloadKeysFlag = false
```

Comment:
why need two flag that value is always the same bro?

Reply:
have updated 👍